docs/python/examples/stock-tracker.md

```diff
@@ -0,0 +1,108 @@
+---
+displayed_sidebar: examplesSidebar
+sidebar_position: 10
+title: Stock Tracker
+---
+
+In this example, we use DBOS to build and deploy an app that tracks stock prices and send SMS alerts when the price of a stock crosses a certain threshold. We certainly don't want to miss a good opportunity to trade, so it's important this workflow is punctual and reliable!
+
+You can see the application live [here](https://max-stock-prices.cloud.dbos.dev/). All source code is [available on GitHub](https://github.com/dbos-inc/dbos-demo-apps/tree/main/python/stock-prices).
```

Comment:
Do the alerts actually work?  Who are they sent to?

Reply:
The SMS are sent to Twilio. You can see them in your twilio account.

The only issue is that, with a free account, they don't get actually sent to the recipient. If you upgrade to a paying account, then they'll go through.